src/LaratrustServiceProvider.php

```diff
@@ -168,8 +168,11 @@ protected function offerPublishing()
         if ($this->app->runningInConsole()) {
             $this->publishes([
                 __DIR__.'/../config/laratrust.php' => config_path('laratrust.php'),
+            ], 'config');
+
+            $this->publishes([
                 __DIR__. '/../config/laratrust_seeder.php' => config_path('laratrust_seeder.php'),
-            ], 'laratrust');
+            ], 'seeder');
```

Comment:
Same here, leave the main config as `laratrust` and then for the seeder `laratrust-seeder`